1.6.1/rockcraft.yaml

```diff
@@ -18,7 +18,7 @@ parts:
     plugin: go
     source: https://github.com/prometheus/pushgateway
     source-type: git
-    source-tag: "v1.6.0"
+    source-tag: "v1.6.1"
     build-snaps:
       - go/1.21/stable
```

Comment:
Just confirming it needs to be 1.21 and not 1.19.

Reply:
I just used the latest stable version of go.
Now I have checked that [pushgateway uses 1.19](https://github.com/prometheus/pushgateway/blob/master/go.mod#L3)